src/common.rs

```diff
@@ -1142,14 +1171,41 @@ pub fn print_transaction_status(
     network_config: &crate::config::NetworkConfig,
 ) -> crate::CliResult {
     eprintln!("--- Logs ---------------------------");
+
+    let price = get_near_price(network_config.coingecko_url.clone())
+        .ok()
+        .flatten();
+    let mut total_gas_burnt = transaction_info.transaction_outcome.outcome.gas_burnt;
+    let mut total_tokens_burnt = transaction_info.transaction_outcome.outcome.tokens_burnt;
+
     for receipt in transaction_info.receipts_outcome.iter() {
+        total_gas_burnt += receipt.outcome.gas_burnt;
+        total_tokens_burnt += receipt.outcome.tokens_burnt;
+
         if receipt.outcome.logs.is_empty() {
             eprintln!("Logs [{}]:   No logs", receipt.outcome.executor_id);
         } else {
             eprintln!("Logs [{}]:", receipt.outcome.executor_id);
             eprintln!("  {}", receipt.outcome.logs.join("\n  "));
         };
     }
+
+    let approximate_usd = if let Some(price) = price {
+        format!("{:.8}", convert_near_to_usd(total_tokens_burnt, price))
+    } else {
+        "N/A".to_string()
+    };
```

Comment:
"(approximate $N/A USD)" is not helpful, so let's remove the whole block from formatting in that case.